### ISSUE TITLE
No Readings at Homekit in Version 0.4.35

### ISSUE THREAD

**Issue by mark82436** [nfarina/homebridge#1698]
i have the problem with th readings from homekit after update from 0.4.33 to 0.4.35
if you change the staus from deconz lights with hardware switch or webinterface
from light off to light on can i not see the status in homekit

after downgrade to version 0.4.33 works it fine
and my automation are working again

**Comment by grover** [comment]
It seems that I introduced this issue, but I do not fully understand how that happens. I've added more logging in https://github.com/KhaosT/HAP-NodeJS/pull/546 and hope that with those logs I'll be able to figure out what I broke.

Sorry about that, but it would be great if you could run the version with additional logging for some time and send me the logs.

Most other users that experience that seem to say that the problem only appears after several hours. Is that the same for you?

**Comment by grover** [comment]
I've forked homebridge to use the version with additional logs. It would be great if someone can assist me by sending me logs of their homebridge installation when the issue occurs. To do that, install the forked version with:

```bash
npm i -g github:grover/homebridge#6bb4b3a99b39daeda980fed49db02209b47bc602
```

(You might need sudo for that.)

and run homebridge with extended logging:

```bash
DEBUG=Accessory,Characteristic,EventedHTTPServer homebridge -D
```

Then once the problem occurs for you I need:

- The log file
- The name of the sensor, accessory, button, ...
- The time when you noticed it

**Comment by NorthernMan54** [comment]
@grover I just started this up in my test environment

PS I used this command line, so I would end up with a file

DEBUG=Accessory,Characteristic,EventedHTTPServer homebridge -D 2>&1 | tee homebridge_debug.log

I'm heading out for a few hours, and look and see if the issue occurs when I'm back

**Comment by grover** [comment]
Thanks!

**Comment by Sonsi79** [comment]
Same Error for me. 
After restart the status update is working fine, after few hours the status isn't updated in HomeKit anymore :-(

Downgrade to 0.4.33 fix this error

**Comment by NorthernMan54** [comment]
This was resolved in 0.4.36.
